README.md

```diff
@@ -266,6 +266,19 @@ application, you should __always__ benchmark if performance matters to you.
 Please visit [Fastify help](https://github.com/fastify/help) to view prior
 support issues and to ask new support questions.
 
+All previous versions of Fastify besides the latest are end of life and will not receive any security or bug fixes.
```

Comment:
This sentence is incorrect. The LTS doc clearly shows that v4 is still supported until 2025-04-30.

Reply:
Changing...